src/auth/auth-config.ts

```diff
@@ -1722,3 +1722,227 @@ export class SmsRegionsAuthConfig {
     }
   }
 }
+/**
+* Enforcement state of reCAPTCHA protection.
+*   - 'OFF': Unenforced.
+*   - 'AUDIT': Create assessment but don't enforce the result.
+*   - 'ENFORCE': Create assessment and enforce the result.
+*/
+export type RecaptchaProviderEnforcementState =  'OFF' | 'AUDIT' | 'ENFORCE';
+
+/**
+* The actions to take for reCAPTCHA-protected requests.
+*   - 'BLOCK': The reCAPTCHA-protected request will be blocked.
+*/
+export type RecaptchaAction = 'BLOCK';
+
+/**
+ * The config for a reCAPTCHA action rule.
+ */
+export interface RecaptchaManagedRule {
+ /**
+  * The action will be enforced if the reCAPTCHA score of a request is larger than endScore.
+  */
+ endScore: number;
+  /**
+  * The action for reCAPTCHA-protected requests.
+  */
+ action?: RecaptchaAction;
+}
+
+/**
+ * The key's platform type: only web is currently supported.
+ */
+export type RecaptchaKeyClientType = 'WEB';
```

Comment:
We actually support ANDROID and IOS too.. 

Does this mean the config returned will not show the mobile platform keys?

Reply:
Nice catch! We do return all 3 keys since we don't validate the key response.  I think only web was supported at the time.  Updated with all 3 type in the new commit.